src/Jackett.Common/Definitions/cpasbienclone.yml

```
@@ -51,7 +52,7 @@
 
   search:
     paths:
-      - path: "{{ if .Keywords }}search_torrent/{{ .Keywords }}{{ .Config.sort }}{{else}}{{end}}"
+      - path: "{{ if .Keywords }}search_torrent/{{ .Keywords }}{{ .Config.sort }}home/{{else}}{{end}}"
```

If it's for empty searches, shouldn't it be like this?
```suggestion
      - path: "{{ if .Keywords }}search_torrent/{{ .Keywords }}{{ .Config.sort }}{{else}}home/{{end}}"
```

---

Crap! Pasted it in the wrong place...

No you're sleep deprived!